### PR TITLE
fix application-connector-validator caching

### PR DESCRIPTION
**Pull request by m00g3n**
**Description**

Changes proposed in this pull request:

- fix cache miss issue

**Related issue(s)**

see also: https://github.com/kyma-project/test-infra/pull/8606

### CODE REVIEW

**Comment by m00g3n** [comment]
-

**Comment by Disper** [comment]
Related PR job https://github.com/kyma-project/test-infra/pull/8606

File: resources/application-connector/charts/central-application-connectivity-validator/values.yaml [hunk 1]
```diff
@@ -37,4 +37,4 @@ istio:
   gateway:
     name: kyma-gateway
     nameMtls: kyma-gateway-application-connector
-    namespace: kyma-system
```
Comment (koala7659) [inline resources/application-connector/charts/central-application-connectivity-validator/values.yaml:40]: Please remove also `cacheExpirationSeconds` and `cacheCleanupIntervalSeconds`

**Review by koala7659 — CHANGES_REQUESTED** [review]
Please check my comment

**Review by akgalwas — APPROVED** [review]

**Review by koala7659 — APPROVED** [review]

**Pull request merged**